roles/nginx/tasks/nginx_apt.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Add nginx key
+  become: yes
+  apt_key:
+    url: http://nginx.org/keys/nginx_signing.key
+    state: present
+  tags: nginx
+
+- name: Add nginx repos
+  become: yes
+  apt_repository:
+    repo: "deb http://nginx.org/packages/ubuntu/ {{ ansible_distribution_release|lower }} nginx"
+    state: present
+    update_cache: yes
```

Comment:
Per #90 we shouldn't `update_cache` for package installs, eg `apt` and `yum` modules, while this is `apt_repostiry` module.

For `apt_repository` module `update_cache: yes` is the default behavior. Optionally this line could be removed to avoid confusion with `apt`/`yum`.


So again: we do `update_cache: yes` when adding new repo (default) and we don't `update_cache: yes` when installing a package (default).